src/css/components/_modals.scss

```diff
@@ -2,16 +2,16 @@
   border-radius: 20px;
   .modal-content {
     border-radius: 10px;
-
+    max-height: 70vh;
```

Comment:
Check the other modals as well. I am sure this is going to create some issue with the othe modals.